deps/ccommon/include/cc_itt.h

```diff
@@ -0,0 +1,71 @@
+#pragma once
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <cc_define.h>
+
+#ifdef CC_ITT
+#include "ittnotify.h"
+
+#define ITT_DOMAIN_NAME "cc_itt"
+
+#define DECLARE_ITT_MALLOC(_name)                                                   \
```

Comment:
`CC_DECLARE_...`

Why only these three macros are capitalized?

Reply:
Done. I used small letters everywhere in my changes like e.g in cc_mm.h module